content/en/blog/_posts/Windows-Memory-Pressure-Eviction.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: blog
+title: "Windows Memory Pressure Eviction"
+slug: windows-memory-pressure-eviction
+canonicalUrl: TBD
+date: TBD
+author: >
+  Mark Rossetti (Microsoft)
+---
+
+In Kubernetes 1.31, support for memory-pressure based eviction was added for Windows nodes.
+
+This post describes what these changes mean for you and how they may differ from usage on Linux nodes.
```

Comment:
I parse this as "This post describes what these changes mean for you and how **these changes** may differ from usage on Linux nodes.". If "they" means something else like "Windows node usage", please be explicit.